src/profile/Signup.js

```diff
@@ -12,93 +15,86 @@ import LoaderButton from "../components/LoaderButton";
  * This component signs you up for the Cognito user
  * @TODO Issue #7
  * 
- * import useState
- *    isLoading: false,
-      email: "",
-      password: "",
-      confirmPassword: "",
-      confirmationCode: "",
-      newUser: null,
+ * import useState x
+ *    isLoading: false, x
```

Comment:
Can you remove these comments and the TODO at the top? Once the issue has been resolved, there is no need to keep this here :)